packages/core/src/odata/common/expandable.ts

```diff
@@ -13,7 +13,4 @@ export type Expandable<EntityT extends EntityBase> = ODataVersion<
   EntityT
 > extends 'v2'
   ? never
-  :
-      | OneToOneLink<EntityT, any>
-      | OneToManyLink<EntityT, any>
-      | AllFields<EntityT>;
+  : OneToOneLink<EntityT, any> | OneToManyLink<EntityT, any>;
```

Comment:
I got this from the reference service:
```
Only navigation properties can be expanded.
```

Reply:
But you can also expand this: `$expand=*`